Doc/library/idle.rst

```diff
@@ -340,9 +340,14 @@ Python Docs
 Turtle Demo
    Run the turtledemo module with example Python code and turtle drawings.
 
-Additional help sources may be added here with the Configure IDLE dialog under
-the General tab. See the :ref:`Help sources <help-sources>` subsection below
-for more on Help menu choices.
+Additional help sources
```

Comment:
IMHO giving more details here is counter-productive. That the menu items must be unique and sorted are rather minor, technical details. And I find this sentence to be stating the obvious: "The allowed file types may depend on the system." As for checking things when submitted, users will discover that themselves when using the feature, and I'm really not sure documenting it here is valuable.